tests/unittests/multimodal/odm/mongo_tests.py

```
@@ -0,0 +1,170 @@
+def fixture_metadata_builder():
+    with patch(
+        "fiftyone.multimodal.odm.mongo.MultimodalMetadata.build_for_scene_inventory"
+    ) as mock:
+        yield mock
+
+
+class ViewFieldMatcher:
+    def __init__(self, expected):
+        self._expected = expected
+
+    def __eq__(self, other):
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__eq__`

(ANN204)